doc/source/overview/algorithms.md

```diff
@@ -14,6 +14,8 @@ The following table summarizes the capabilities of the current algorithms:
 |Method|Models|Exp. types|Classification|Regression|Tabular|Text|Image|Cat. data|Train|Dist.|
 |:---|:---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---|:---:|
 |[ALE](../methods/ALE.ipynb)|BB|global|✔|✔|✔| | | |✔| |
+|[PartialDependence](../methods/PartialDependence.ipynb)|BB WB|global|✔|✔|✔| | |✔|✔| |
+|[PartialDependenceVariance](../methods/PartialDependenceVariance.ipynb)|BB WB|global|✔|✔|✔| | |✔|✔| |
```

Comment:
Unfortunately with this line the content of the page extends beyond the formatted page: 
![image](https://user-images.githubusercontent.com/13080878/193861693-010dafeb-a772-4a1f-848e-b7ea00503a61.png)
One fix would be to increase the width of the white part in the css style page. Another I would suggest to shorten the names of the explainers, e.g. `PartialDependence` -> `Partial Dependence` and `PartialDepencenceVariance` -> `PD Variance` since the names in this table are descriptive, not exact class names.